### PR TITLE
Implemented teaser edit

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/massiveart/husky/pull/705
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR add the ability to edit teasers locally on the page where they are choosen.

#### TODO

- [x] Test for MediaRedirectController
- [x] Styling change media button
- [x] Translations

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Teaser/TeaserManager.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        if (0 === count($items)) {
+            return [];
+        }
+
+        $result = [];
+        foreach ($this->sort($items) as $type => $typeIds) {
+            $teasers = $this->providerPool->getProvider($type)->find($typeIds, $locale);
+
+            foreach ($teasers as $teaser) {
+                $index = array_search(['type' => $type, 'id' => $teaser->getId()], $items);
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Teaser/TeaserManager.php]: sorting not working if additional properties selected.

**Comment by chirimoya** [comment]
- reset removes the preview image

File: src/Sulu/Bundle/ContentBundle/Teaser/Teaser.php [hunk 2]
```diff
@@ -174,4 +174,43 @@ public function getAttributes()
...
+    {
+        return new self(
+            $this->id,
+            $this->type,
+            $this->locale,
+            $this->getValue('title', $item, $this->getTitle()),
+            $this->getValue('description', $item, $this->getDescription()),
+            $this->getValue('moreText', $item, $this->getMoreText()),
+            $this->getValue('url', $item, $this->getUrl()),
+            $this->getValue('mediaId', $item, $this->getMediaId())
```
Comment (chirimoya) [inline src/Sulu/Bundle/ContentBundle/Teaser/Teaser.php]: missing attributes

File: src/Sulu/Bundle/MediaBundle/Resources/config/routing.yml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+    path: /template/collection/new.html
     defaults: { _controller: SuluMediaBundle:Template:collectionNew }
 
 sulu_media.template.media.multiple_edit:
-    pattern: /template/media/multiple-edit.html
+    path: /template/media/multiple-edit.html
     defaults: { _controller: SuluMediaBundle:Template:multipleEdit }
+
+sulu_media.redirect:
+    path: /{id}
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Resources/config/routing.yml]: would add redirect to url

**Comment by chirimoya** [comment]
tests are failing (media bundle)

**Pull request merged**